did_core/did_methods/did_resolver_sov/src/resolution/utils.rs

```diff
@@ -53,6 +53,29 @@ fn unix_to_datetime(posix_timestamp: i64) -> Option<DateTime<Utc>> {
         .map(|date_time| DateTime::<Utc>::from_naive_utc_and_offset(date_time, Utc))
 }
 
+fn expand_abbreviated_verkey(did: &str, verkey: &str) -> Result<String, DidSovError> {
+    if let Some(stripped_key) = verkey.strip_prefix('~') {
+        let stripped_did = &did[8..];
```

Comment:
bit of a nit pick;

i'd refer if we referred to "stripped_did" as the "nym". i think that term is more widely understood.

also, altho `did[8..]` would work to remove the `did:sov` prefix, it isn't very clear that this is what is happening here.

IMO it would be nice if `expand_abbreviated_verkey` just took in `nym` rather than `did`.

then where `expand_abbreviated_verkey` is called, you'll need to pre-calculate the `nym` from the `did`. you might be able to use the `ddo_id` 
`Did` struct for this actually; it should have a `.id()` method, which will return the end-part of the DID (aka the NYM)